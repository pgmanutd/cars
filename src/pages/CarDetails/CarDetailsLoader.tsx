import React from 'react';
import Box from '@material-ui/core/Box';

const CarDetailsLoader: React.FC = () => (
  <Box component="section" data-testid="CarDetailsLoader">
    Loading...
  </Box>
);

export default CarDetailsLoader;
