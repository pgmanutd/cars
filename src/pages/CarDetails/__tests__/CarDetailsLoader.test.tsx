import React from 'react';
import { render } from '@testing-library/react';

import CarDetailsLoader from '../CarDetailsLoader';

describe('<CarDetailsLoader />', () => {
  const setup = () => {
    const renderResult = render(<CarDetailsLoader />);

    return {
      renderResult,
    };
  };

  it('should render the component and matches it against stored snapshot', () => {
    const { renderResult } = setup();

    expect(renderResult.asFragment()).toMatchSnapshot();
  });
});
