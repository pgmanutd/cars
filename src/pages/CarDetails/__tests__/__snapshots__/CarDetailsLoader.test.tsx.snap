// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CarDetailsLoader /> should render the component and matches it against stored snapshot 1`] = `
<DocumentFragment>
  <section
    data-testid="CarDetailsLoader"
  >
    Loading...
  </section>
</DocumentFragment>
`;
